tests/natural_language/SpellCheckBearTest.py

```diff
@@ -0,0 +1,17 @@
+import platform
+import unittest
+
+from bears.natural_language.SpellCheckBear import SpellCheckBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = "This is correct spelling."
+
+bad_file = "tihs si surly som incoreclt speling."
+
+
+SpellCheckLintBearTest = unittest.skipIf(
```

Comment:
This had to be done because it doesn't work on windows - https://github.com/myint/scspell/issues/15